dsf-fhir/dsf-fhir-server/src/main/resources/db/db.read_access.changelog-0.6.0.xml

```diff
@@ -52,6 +52,10 @@
 				UNION
 				SELECT provenance_id AS id, version, 'Provenance'::text AS type, provenance AS resource FROM current_provenances
 				UNION
+				SELECT questionnaire_id AS id, version, 'Questionnaire'::text AS type, questionnaire AS resource FROM current_questionnaires
```

Comment:
Changes in this file may result in different change set hash, DB upgrade 0.7.0 -> 0.8.0 may not work. A new `db.read_access.changelog-0.8.0.xml` might be needed with the content of this file.